modules/backup-gcs/client.go

```diff
@@ -324,23 +290,23 @@ func (g *gcsClient) Write(ctx context.Context, backupID, key string, r io.ReadCl
 	return written, nil
 }
 
-func (g *gcsClient) Read(ctx context.Context, backupID, key string, w io.WriteCloser) (int64, error) {
+func (g *gcsClient) Read(ctx context.Context, backupID, key, overrideBucket, overridePath string, w io.WriteCloser) (int64, error) {
 	defer w.Close()
 
-	bucket, err := g.findBucket(ctx)
+	bucket, err := g.findBucket(ctx, overrideBucket)
 	if err != nil {
-		err = fmt.Errorf("find bucket: %w", err)
+		err = fmt.Errorf("read cannot:%w", err)
```

Comment:
suggestion: maybe `read: find bucker: %w` message would be better here?

Reply:
done